pkgs/servers/mautrix-telegram/default.nix

```diff
@@ -1,30 +1,18 @@
-{ lib, fetchpatch, python3 }:
+{ lib, fetchFromGitHub, python3 }:
 
 with python3.pkgs;
 
 buildPythonPackage rec {
   pname = "mautrix-telegram";
-  version = "0.4.0.post1";
+  version = "0.5.0-rc3";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "7a51e55a7f362013ce1cce7d850c65dc8d4651dd05c63004429bc521b461d029";
+  src = fetchFromGitHub {
```

Comment:
Why not fetchPypi?

Reply:
Because the RC are not pushed to pypi. I made this branch only to test the RC before the final version is out. Once the new version is on Pypi I will change this to fetchPypi again.